test_agent_api.py

```
@@ -0,0 +1,164 @@
+import time
+
+BASE_URL = "http://localhost:10000"
+
+def test_agent_api():
+    """Test all agent management API endpoints"""
+    print("🧪 Testing Agent Management API Endpoints\n")
+
+    # Test health endpoint first
+    print("1. Testing health endpoint...")
+    try:
+        response = requests.get(f"{BASE_URL}/health")
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)